connector.rb

```diff
@@ -2613,7 +2658,7 @@
               person_nodeID: data['person_nodeID'],
               person_name: data['person_name'],
               instance_name: data['instance_name'],
-              data: data['payload'],
+              data: data['payload'].map { |key, value| { 'key' => key, 'value' => value } },
```

Comment:
@klaasvandeweerdt could you highlight the differences in output? Would this impact existing recipes that use this?

Reply:
@bennettgo, the raw data is a key/value pair with an unknown number of fields and data types. The initial design was to expose that raw data as it is. This allows me to easily check for specific fields, loop through those fields, etc. It makes it easy to update specific fields using formulas and post back the whole array as such.
Next to that, I can also define a mapping of all those fields to a data pill via the `schema-designer`.